usr/src/man/man1m/zpool.1m

```diff
@@ -77,7 +78,7 @@
 .Nm
 .Cm import
 .Op Fl D
-.Op Fl d Ar dir
+.Op Fl c Ar cachefile Ns | Ns Fl d Ar dir
```

Comment:
I'm not sure if "-c cachefile" was left out on purpose here.  If that's not the case, please make sure to update this form of "import" subcommand in the DESCRIPTION as well.